tests/compute.py

```diff
@@ -359,3 +360,70 @@ def compute_image_load_store(cfg: SceneCfg, show_dbg_points=False):
         group.add_children(get_debug_points(cfg, cuepoints))
 
     return group
+
+
+_IMAGE_3D_STORE_COMPUTE = """
+void main()
+{
+    ivec2 pos = ivec2(gl_LocalInvocationID.xy);
+    float size = float(imageSize(texture).x);
+    float color = (float(pos.x) + float(pos.x * pos.y)) / (size * size) ;
```

Comment:
I think you want `float color = (float(pos.y) * size + float(pos.x)) / (size * size);` here

Reply:
Thanks. Done.

Reply:
you left a space before the `;`

Reply:
Fixed.